src/components/optimizerTab/OptimizerSuggestionsModal.tsx

```diff
@@ -457,3 +511,59 @@ export function ZeroResultSuggestionModal() {
     </Modal>
   )
 }
+
+type Request = {
```

Comment:
Is this the same as `import { Form } from 'types/Form'`?

Reply:
oh probably (: didn't realise we had a type for the form